### ISSUE TITLE
Grouping + sum , groupingRows question

### ISSUE THREAD

**Issue by turbaceanu15** [mbrn/material-table#2815]
# Guidelines

- #### Please include a demo of the issue/behavior/question you have
![image](https://user-images.githubusercontent.com/72971720/110627777-0ba9cc00-81ab-11eb-8583-9cb0adf6602a.png)
So here I have a table where I used grouping by a hidden column called Name,
As you see I have something like a calendar, i would like to know if it is possible to summary the values from every cell and render it up near the record with grouping feature.

- #### Please try to be as detailed as possible

- Something like that :
![image](https://user-images.githubusercontent.com/72971720/110628092-62afa100-81ab-11eb-9eb6-652cae7a36ea.png)
A record with a sum of everything from that grouped rows, something like the ones from expended panel but just one record, it is possible somehow to do this ?
Basically , it`s a sum of everything per day.

Thanks in advance.
